source/wp-content/themes/wporg-showcase-2022/functions.php

```diff
@@ -154,3 +156,37 @@ function modify_excerpt_length() {
 function modify_excerpt_more() {
 	return '...';
 }
+
+/**
+ * Modify query vars for mast head query.
+ *
+ * See: https://github.com/WordPress/gutenberg/issues/41184
+ *
+ * @return string
+ */
+function modify_query_loop_block_query_vars( $query, $block ) {
+	if ( isset( $block->context['query']['sticky'] ) && 'only' === $block->context['query']['sticky'] ) {
+		$sticky = get_option( 'sticky_posts' );
+		$query['ignore_sticky_posts'] = 1;
+		$query['post__in'] = $sticky;
+	}
+
+	return $query;
+}
+
+/**
+ * Add SVGs to the list of allowed mime types.
+ *
+ * @param array $upload_mimes The existing allowed mime types.
+ *
+ * @return array The enhanced allowed mime types.
+ */
+function wporg_add_svg_to_upload_mimes( $upload_mimes ) {
+	if ( current_user_can( 'manage_options' ) ) {
```

Comment:
@dd32 Does this make any sense? To allow SVG uploads but lock it down to admins?

Reply:
In all honesty, this doesn't belong in the theme, and I'm not sure the context of adding it.

There's also something to say that this should be using `unfiltered_upload`, but that isn't available to anyone on WordPress.org.

I think I'd rather enable a plugin such as https://wordpress.org/plugins/safe-svg/ (or one of the others) on the site instead if SVGs are required.

Reply:
Yeah fair, I'll avoid adding SVG support.

Reply:
I think we'll need SVG support eventually, so I've added a low-priority issue over in mu-plugins: https://github.com/WordPress/wporg-mu-plugins/issues/300

Reply:
Thanks!